### DISCUSSION TITLE
Has hypridle been not working for anybody?

### DISCUSSION THREAD

**Discussion by NextStep-IM** [end-4/dots-hyprland#365] [Q&A]
`hypridle-git` stopped working a while ago and gives this error: `hypridle: error while loading shared libraries: libhyprlang.so.1: cannot open shared object file: No such file or directory`. I installed the one in the `extra` repo and it ran but didn't trigger hyprlock.

**Comment by Alanzcc** [comment]
I found a fix on reddit: https://www.reddit.com/r/hyprland/s/YaWj5BcAgH
The most upvoted answer fixes it by creating an alias.
However I can't get hypridle timeout to work
